pallets/utxo/src/lib.rs

```diff
@@ -386,7 +521,7 @@ pub mod pallet {
         trx.encode()
     }
 
-    fn disperse_reward<T: Config>(auths: &[H256], block_number: T::BlockNumber) {
+    fn reward_tx_validation<T: Config>(auths: &[H256], block_number: T::BlockNumber) {
```

Comment:
Maybe a better name for this function would be `reward_block_validation`?
In fact, `disperse_reward` wasn't so bad. Also `grant_block_reward` sounds good.